### DISCUSSION TITLE
Removing runner from host *after* force removal in github

### DISCUSSION THREAD

**Discussion by akiross** [community/community#160502] [Actions] [answered]
Hello,
I made a mistake and force removed an action runner from github thinking I had no access to the machine, but I actually removed the wrong runner. So, right now I have a Linux host with a runner installed, but not registered in github.

If I run `./config --url https://github.com/MY_ORG --token THE_OLD_TOKEN` I get this error:

> Cannot configure the runner because it is already configured. To reconfigure the runner, run 'config.cmd remove' or './config.sh remove' first.

If I try to remove it via `./config remove`, I am asked a remove token, but I don't have it (anymore).

Is there a way to force the removal and/or re-register the runner using the old token?
Thank you

**Comment by freshusername** [comment]
I have the same issue. Does anybody know how to solve it?

**Reply by MoJo2600** [reply, marked as answer]
You can remove the `.runner` file and then execute `./config.sh remove`. After that your runner can be registered again with a new token.

**Reply by freshusername** [reply]
Thank you, this helps. I also fixed it by simply removing actions-runner folder and installing it from scratch

**Reply by AlirezaBahadoriZadeh2002** [reply]
Thanks a lot
it worked !!

**Reply by hogimn** [reply]
Thanks~!

**Reply by huberthomas** [reply]
Big thanks! Additionally in Windows the file is hidden in the github-runner directory! So, ...

- just enable view hidden files in the file explorer to remove it per click or 
- use powershell directly in the github-runner directory with `rm .runner`

**Reply by javiBajoCero** [reply]
this worked for me, just rm .runner and then 
![image](https://github.com/actions/runner/assets/25673527/8c55bb4b-482e-46ed-bec8-761e4bd79f0e)

**Reply by IronSean** [reply]
> You can remove the `.runner` file and then execute `./config.sh remove`. After that your runner can be registered again with a new token.

And where does one find this file to remove?

**Reply by huberthomas** [reply]
Hi, it is typically all located in the Github runner directory, but I guess the .runner file only exists if there was a running instance before. Best regards!

**Reply by AniMB** [reply]
Thanks a lot!! Perfect

**Reply by androetto** [reply]
Great!

**Reply by CDTR-MattConroy** [reply]
This doesn't seem to work anymore:
```
PS C:\actions-runner> .\config.cmd remove

# Runner removal

Failed: Removing runner from the server
Value cannot be null. (Parameter 'configuredSettings')
```

**Reply by luke-z** [reply]
I also deleted `.runner_migrated` then it worked

**Reply by codeskipper** [reply]
> I also deleted `.runner_migrated` then it worked

### this worked for me too

**Reply by FPA-JohnHutchinson** [reply]
After removing `.runner` I get another error:

```
Failed: Removing runner from the server
Value cannot be null. (Parameter 'configuredSettings')
```

I then removed `.runner_migrated`, which then allowed `./config.sh remove` to be successful.

**Reply by CDTR-MattConroy** [reply]
I have no `.runner` and no `.runner_migrated` file. I get the same error as above
```
# Runner removal

Removing service
Failed: Removing service
Value cannot be null. (Parameter 'configuredSettings')
```

Can we get a clearer error of what's going wrong or how to correct it?

**Comment by codewitgabi** [comment]
Big thanks to you man.

**Comment by ShruB1820** [comment]
I have the same issue. Once I execute `./config.sh remove` command its asking me to give **runner removal token**. But there is no runner inside the action -> runners. I'm confused here. Please help me asap.
